tests/problem3.test.js

```diff
@@ -24,6 +24,17 @@
  */
 
 const p = require('../utils.js');
+const first = async (promises) => {
```

Comment:
Niestety popełniliśmy błędy w testach do tego zadania. Jeśli domerdżujesz sobie naszego mastera, testy przestaną przechodzić. :) To nie jest rozwiązanie, które mieliśmy na myśli.